lib/fizzy/limits.hpp

```diff
@@ -18,6 +18,7 @@ static_assert(MaxMemoryPagesLimit == 65536);
 
 /// The default hard limit of the memory size (256MB) as number of pages.
 constexpr uint32_t DefaultMemoryPagesLimit = (256 * 1024 * 1024ULL) / PageSize;
+static_assert(DefaultMemoryPagesLimit == 4096);
```

Comment:
Why this is defined twice then?

Reply:
Because it doesn't worth to introduce a dependency form C++ to C API only for this yet.

Reply:
Can add a `static_assert(::DefaultMemoryPagesLimit == fizzy::DefaultMemoryPagesLimit)` in `capi_test.cpp`?

Reply:
> Can add a `static_assert(::DefaultMemoryPagesLimit == fizzy::DefaultMemoryPagesLimit)` in `capi_test.cpp`?

Added.

Reply:
> Because it doesn't worth to introduce a dependency form C++ to C API only for this yet.

I don't agree here. This case returns from time to time.